controllers/rcj_soccer_referee_supervisor/referee/penalty_area_checker.py

```diff
@@ -0,0 +1,73 @@
+from typing import List
+
+from referee.consts import BLUE_PENALTY_AREA, YELLOW_PENALTY_AREA
+
+
+class PenaltyAreaChecker:
+    def __init__(self, allowed: int, reset_after: int, time: int):
+        self.allowed = allowed
+        self.reset_after = reset_after
+        self.time = time
```

Comment:
@Adman let me see if I get this right, this is an attempt at passing a reference of the "current time" from the parent `Supervisor`?

Reply:
Yes. But it seems useless since the time is remembered in `track` method. Will fix that.